### PR TITLE
Enable press and hold for notation tabs scroll buttons

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #11864

(While looking at IncrementalPropertyControl.qml for a quick reference, I found that there were some QML warnings in that file so I quickly fixed those too.)

### CODE REVIEW

File: src/notation/qml/MuseScore/NotationScene/internal/NotationSwitchScrollArrowButton.qml [hunk 1]
```diff
@@ -26,6 +26,8 @@ import MuseScore.UiComponents 1.0
 FlatButton {
     id: root
 
+    signal scrollRequested
```
Comment (Eism) [inline src/notation/qml/MuseScore/NotationScene/internal/NotationSwitchScrollArrowButton.qml:29]: Please add round brackets

File: src/framework/uicomponents/qml/MuseScore/UiComponents/IncrementalPropertyControl.qml [hunk 2]
```diff
@@ -49,7 +49,7 @@ Item {
     implicitHeight: 30
     implicitWidth: parent.width
 
-    navigation.name: root.objectName != "" ? root.objectName : "IncrementalControl"
+    navigation.name: root.objectName ?? "IncrementalControl"
```
Comment (Eism) [inline src/framework/uicomponents/qml/MuseScore/UiComponents/IncrementalPropertyControl.qml:52]: My opinion about '??'
https://github.com/musescore/MuseScore/pull/11314#pullrequestreview-957450120

I suggest not to use '??'

Comment (cbjeukendrup) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/IncrementalPropertyControl.qml]: Okay, I reverted it to the `Boolean(...) ? ... : ...` solution. I think we shouldn't let a buggy IDE limit which language features we use though...

**Comment by DmitryArefiev** [comment]
Tested #11864 on Win10, Mac12, LinuxMint 20.2 - **DONE**

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**